doc/source/whatsnew/v0.19.0.txt

```diff
@@ -439,6 +439,7 @@ API changes
 - ``Timestamp.to_pydatetime`` will issue a ``UserWarning`` when ``warn=True``, and the instance has a non-zero number of nanoseconds (:issue:`14101`)
 - ``Panel.to_sparse`` will raise a ``NotImplementedError`` exception when called (:issue:`13778`)
 - ``Index.reshape`` will raise a ``NotImplementedError`` exception when called (:issue:`12882`)
+- ``pd.read_csv()``, ``pd.read_table()``, and ``pd.read_hdf()`` raise a ``FileNotFoundError`` exception for Python >= 3.3 when called on a nonexistent file (:issue:`14086`)
```

Comment:
update this to say that this is the standard for file not found and is back-compat to OSError.